app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_likes_counter
+
+  private
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
```

Comment:
[OPTIONAL]

Hi, Just like your post model, it'll be better to use the `post.increment!` method instead here::

- You are using the `update` method with the `count` method. This is inefficient and prone to errors, as it requires an extra database query and does not handle concurrency issues. I kindly suggest that you make use of the increment! method to update the likes counter for a post
- The `increment!` method is a built-in method in Rails that increases the value of an attribute by one and saves the record. It also uses optimistic locking to prevent race conditions. You can read more about it [here](https://apidock.com/rails/ActiveRecord/Base/increment!).

- Here is an example of how your code should look like:

```ruby
# app/models/like.rb
...

def update_likes_counter
  post.increment!(:likes_counter)
end
```

I hope this helps you improve your code. Please let me know if you have any questions or feedback. Happy coding! 😊